sections/annoyances.txt

```diff
@@ -30,6 +30,12 @@ ado.hu###cookieBox
 ado.hu###cookieBox_bg
 adoforum.hu###cookies-alert
 adozona.hu##.floating-help-wrapper
+alza.hu##.chat-wrapper
```

Comment:
It seems to me that the chat is part of the site's functions, I don't think it should be blocked

Reply:
You just hide a chat. It is not blocked in this case.